wcfsetup/install/files/lib/system/session/SessionHandler.class.php

```diff
@@ -991,4 +984,140 @@ protected function getSpiderID($userAgent) {
 	public function isFirstVisit() {
 		return $this->firstVisit;
 	}
+	
+	/**
+	 * Returns all user sessions for a specific user.
+	 *
+	 * @return      \wcf\system\session\Session[]
+	 * @since       5.4
+	 */
```

Comment:
```suggestion
	 * @return      \wcf\system\session\Session[]
	 * @throws      InvalidArgumentException if the given user is a guest.
	 * @since       5.4
	 */
```